test/md022-line-number-out-of-range.md

```diff
@@ -1 +1 @@
---
\ No newline at end of file
+--
```

Comment:
If I remember, this file’s contents needed to be exactly as they were to reproduce the issue. Could you please use a JSON file to disable 047 for this file instead?